frontend/benefit/applicant/src/components/applications/forms/application/deMinimisAid/list/DeMinimisAidsList.tsx

```diff
@@ -42,7 +42,8 @@ const DeMinimisAidsList: React.FC = () => {
               {grant[DE_MINIMIS_AID_KEYS.GRANTER]}
             </$GridCell>
             <$GridCell $colSpan={2}>{`${
-              grant[DE_MINIMIS_AID_KEYS.AMOUNT] ?? ''
+              grant[DE_MINIMIS_AID_KEYS.AMOUNT]?.toString().replace('.', ',') ??
+              ''
```

Comment:
Why don't we use here a util function?

Reply:
they are abit different. Opposite transform directions, but i can write the other one;)